parsec/core/types/pki.py

```diff
@@ -1,210 +1,9 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
-from __future__ import annotations
 
-from pathlib import Path
-from typing import Dict, Iterable, Union
+from parsec._parsec import LocalPendingEnrollment, PkiEnrollmentLocalPendingError, X509Certificate
 
-import attr
-
-from parsec._parsec import DateTime, EnrollmentID
-from parsec.api.data import DataError, PkiEnrollmentSubmitPayload
-from parsec.api.protocol.pki import EnrollmentIDField
-from parsec.core.local_device import MsgpackSerializer
-from parsec.core.pki.exceptions import (
-    PkiEnrollmentLocalPendingError,
-    PkiEnrollmentLocalPendingNotFoundError,
-    PkiEnrollmentLocalPendingPackingError,
-    PkiEnrollmentLocalPendingValidationError,
-)
-from parsec.core.types.backend_address import (
-    BackendPkiEnrollmentAddr,
-    BackendPkiEnrollmentAddrField,
-)
-from parsec.core.types.base import BaseLocalData
-from parsec.serde import BaseSchema, fields, post_load
-
-
-@attr.s(slots=True, frozen=True, auto_attribs=True)
-class X509Certificate:
-    class SCHEMA_CLS(BaseSchema):
-        type = fields.CheckedConstant("x509_certificate", required=True)
-        issuer = fields.Dict(required=True)
-        subject = fields.Dict(required=True)
-        der_x509_certificate = fields.Bytes(required=True)
-        certificate_sha1 = fields.Bytes(required=True)
-        certificate_id = fields.String(required=True, allow_none=True)
-
-        @post_load
-        def make_obj(self, data: Dict[str, Union[str, Dict[str, str], bytes]]) -> X509Certificate:
-            data.pop("type", None)
-            return X509Certificate(**data)  # type: ignore[arg-type]
-
-    issuer: Dict[str, str]
-    subject: Dict[str, str]
-    der_x509_certificate: bytes
-    certificate_sha1: bytes
-    certificate_id: str | None
-
-    def is_available_locally(self) -> bool:
-        """Certificates that are received from another peer are not available locally."""
-        return self.certificate_id is not None
-
-    @property
-    def subject_common_name(self) -> str | None:
-        return self.subject.get("common_name")
-
-    @property
-    def subject_email_address(self) -> str | None:
-        return self.subject.get("email_address")
-
-    @property
-    def issuer_common_name(self) -> str | None:
-        return self.issuer.get("common_name")
-
-
-class PendingDeviceKeys(BaseSchema):
-    type = fields.CheckedConstant("requested_device_keys", required=True)
-    private_key = fields.PrivateKey(required=True)
-    signing_key = fields.SigningKey(required=True)
-
-
-pending_device_keys_serializer = MsgpackSerializer(
-    PendingDeviceKeys,
-    validation_exc=PkiEnrollmentLocalPendingValidationError,
-    packing_exc=PkiEnrollmentLocalPendingPackingError,
+__all__ = (
```

Comment:
I'm wondering if it's a good thing to re-export rust objects like this, instead of editing related imports statements. Importing from `_parsec` makes it clear that it's not a pure python object thus it helps while debugging. What do you think?

Reply:
It breaks `parsec-ext` where I can't access